pages/base.py

```diff
@@ -47,57 +47,44 @@ def __getattr__(self, what):
 class BasePage(BaseElement):
     url = None
 
-    def open(self):
+    def goto(self):
         self.driver.get(self.url)
 
+    def verify_page(self):
+        """
+        Use an element from the locator_dictionary to check if we're on expected page
+        """
+        try:
+            self.__getattr__(list(self.locator_dictionary.keys())[0])
```

Comment:
As discussed, it would be great if this element is chosen carefully. I think we might run into issues verifying with urls when testing ember pages as sometimes the url is the same (happens in SPA) but you are at a different logical page.